client/state/action-types.js

```diff
@@ -330,6 +330,11 @@ export const HELP_TICKET_CONFIGURATION_REQUEST_FAILURE =
 export const HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS =
 	'HELP_TICKET_CONFIGURATION_REQUEST_SUCCESS';
 export const HISTORY_REPLACE = 'HISTORY_REPLACE';
+export const HOSTING_CREATE_SFTP_USER = 'HOSTING_CREATE_SFTP_USER';
+export const HOSTING_RECEIVE_SFTP_USER = 'HOSTING_RECEIVE_SFTP_USER';
```

Comment:
Calypso does have a convention to move verbs to the end, mainly for alphabetization. (It does read awkwardly).

Would recommend one `HOSTING_SFTP_USER_CREATE` action, as it's good to avoid `RECEIVE` actions. In context of the app it's kind of meaningless, even though we mean to dump API state into the global store.